### ISSUE TITLE
Error recognizing nonstandard residue names

### ISSUE THREAD

**Issue by skandel12** [prody/ProDy#254]
I work mostly with simulation data with PDBs in Amber format, and I have been using ProDy to do the bulk of my analysis. Quite problematically, I just realized that ProDy does not recognize nonstandard residue names in Amber format. For instance, if I have a PDB p that contains the residue HID (instead of HSD):
-  p.select('protein') excludes the residue HID
-  p.select('calpha') excludes the calpha atom in the residue HSD
- even p.select('resname HID').select('calpha') returns nothing.

I tried the residues HID, HIS, and CYX, and had the same issue for all of them. I don't know if there are other such residues. 

Would it be possible to fix the issue?

**Comment by cihankayacihan** [comment]
Hi. 

Thanks for feedback. We will try to fix this issue in next release. Thanks.

**Comment by duming** [comment]
I have the same problem, when I try to use Prody to parse CATH database(V4.0.0).
There are 31275/235840 domains can not be parsed properly.
In the worst case, there are only 8 atoms been recognize out of 746 atoms.
Worse case domain id: 4eqjA02.

**Comment by SHZ66** [comment]
Hi,

ProDy does handle the non-standard residue names, but somehow HID and CYX are not in the list. Now it has been fixed in #338.

In case you want to add more such residues name, you can use following example code:
```
import prody.atomic.flags as flags
flags.NONSTANDARD['HID'] = set(['large', 'polar', 'basic', 'surface', 'cyclic'])
print flags.NONSTANDARD
```

Please note that the properties in the set affect the selection behavior. Thanks for the input!
